src/editor/CodeHintManager.js

```diff
@@ -249,7 +249,8 @@ define(function (require, exports, module) {
         hintList         = null,
         deferredHints    = null,
         keyDownEditor    = null,
-        codeHintsEnabled = true;
+        codeHintsEnabled = true,
+        codeHintOpened   = false;
```

Comment:
Created a boolean to make sure that when you press ctrl+space (the default key map for hint suggestions) it doesn't close it after first time it's called.